sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdConstants.java

```diff
@@ -598,7 +598,8 @@ public enum RntbdRequestHeader implements RntbdHeader {
         SDKSupportedCapabilities((short) 0x00A2, RntbdTokenType.ULong, false),
         ChangeFeedWireFormatVersion((short) 0x00B2, RntbdTokenType.String, false),
         PriorityLevel((short) 0x00BF, RntbdTokenType.Byte, false),
-        GlobalDatabaseAccountName((short) 0x00CE, RntbdTokenType.String, false);
+        GlobalDatabaseAccountName((short) 0x00CE, RntbdTokenType.String, false),
+        ThroughputBucket((short)0x00DB, RntbdTokenType.Byte, false); //TODO: does the throughput bucket supported in thin client
```

Comment:
Thin client is a good question. Also GAteway mode is relevant - we would rely on ComputeGateway and Routing Gateway to pass through the header. I would expect this to work in thin client because the rntbd body is passed through opaquely.